hangb.rb

```diff
@@ -0,0 +1,262 @@
+# Hangman
+require "colorize" # Colorize gem implementation used to change the different pieces of
+                   # the ship different colors in accordance with wrong guesses.     
+
+class Hangman       # Creation of class, always capitalized 
+  
+  # (def(create method) initiate ) 
+  # m: " Hey Universe , I want to create a program (cake) called Hangman.(Thus the capital "H" above.)
+  # Universe: okay. cool, m. What do you want in this program. What variables (ingredients) 
+  #           do you need to make this program (recipe)???
+  # m: " well, I know for sure I'll need an answer, an answer line, and I need to keep track 
+  #     of wrong guesses. I'm gonna create a method for how to start the game. could you keep 
+  #      track of that as well?"
+  # Universe: cool. put an at (@) in front of those variables initially, just so I know to be on 
+  #           the lookout for those; 
+  #           so I can "instantiate"/ thunderbolt, lightning, zap/create those for you.
+  # m: "thanks, Lady!"
+  # Universe: No problem, my love. 
+  #           Remember though, you're gonna need to give those names when you ask for them initially.
+  #           So we're in the same loop.
+  # m: "U got jokes!"" *smirk emoji*
+  # Universe: Girl, always.   
+
+
+
+  def initialize     
+    # Answer which will be pulled from random list of 10 words
+    @answer = populate_answer
+    # Display
+    @answer_line = answer_line
+    # count of wrong answers
+    @wrong_count = 0
+    start
+  end
+
+  def start   # opening screen: puts greeting, full Enterprise visible in default terminal color.
+    puts "Welcome to Star Trek Hangman! Good luck!"
+    # Keeps track of game play
+    @now_playing = true  
+
+    draw_board
+    answer_line
+    # play
+    get_guess
+  end
+
+# ------------------
+# Create answer
+
+# m: "U, this is the method to create @answer." 
+# U:  yup, yup.
+# m:" here are the answers for the game. I put them in an array, U. That way, you have an assigned
+#     value for each one. Please populate the answer by shuffling the array then pop the last word out 
+#     after you shuffle the array. I'm gonna use that word as the "answer" each time. "return" means 
+#     give it back to me after you shuffle.pop"
+# U:  Word. got it.
+
+
+  def populate_answer           
+    answer_array = ["PICARD", "GUINAN", "WORF", "WARP", "ENGAGE", "ENTERPRISE", "DATA", "ROMULAN", "VULCAN", "BETAZOID", "KLINGON"]
+
+    answer = answer_array.shuffle.pop
```

Comment:
For an easier way, arrays have a built in method to get a random element called [`#sample`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-sample).

You could also shortcut this by doing:
```
   return answer_array.shuffle.pop
```